libgeopmd/src/DrmSysfsDriver.cpp

```diff
@@ -22,6 +21,57 @@
 static const std::string DRM_DIRECTORY = "/sys/class/drm";
 static const std::string GPU_CARD_PREFIX = "card";
 static const std::string GPU_TILE_PREFIX = "gt";
+static const std::string HWMON_PREFIX = "hwmon";
+static const std::string HWMON_NAME_CARD = "i915\n";
+static const std::string HWMON_NAME_TILE_PREFIX = "i915_gt";
+static const std::string GPU_SIGNAL_NAME_SUFFIX = "::GPU";
+static const std::string TILE_SIGNAL_NAME_SUFFIX = "::GPU_CHIP";
+
+static std::map<std::pair<geopm_domain_e, int>, std::string> map_geopm_index_to_hwmon_path(const std::string &drm_directory, int num_tiles_per_gpu)
```

Comment:
Seems like we should be able to derive the PlatformTopo information about GPUs from this class map.  Maybe we could wrap the map in an accessor class and reuse it in the PlatformTopo imp?

Reply:
Opened #3559